pimd/pim_vty.c

```diff
@@ -354,6 +355,17 @@ static int pim_igmp_config_write(struct vty *vty, int writes,
 
 	return writes;
 }
+#else
+static int gm_config_write(struct vty *vty, int writes,
+			   struct pim_interface *pim_ifp)
+{
+	if (pim_ifp->mld_version != MLD_DEFAULT_VERSION)
+		vty_out(vty, " ipv6 mld version %d\n", pim_ifp->mld_version);
+	if (pim_ifp->gm_default_query_interval != IGMP_GENERAL_QUERY_INTERVAL)
+		vty_out(vty, " ipv6 mld query-interval %d\n",
```

Comment:
Need to add other CLIs show running as well here :
1. ipv6 mld join X:X::X:X$group
2. ipv6 mld
3. ipv6 mld query-max-response-time (1-65535)
etc, etc

Reply:
> `ipv6 mld join X:X::X:X$group`

This one is completely separate code and does not belong in this PR.

> ipv6 mld
> ipv6 mld query-max-response-time (1-65535)

These do belong here, but I'm going to say it's not necessary to have it in the same PR. This PR is already relatively large, and the question to evaluate here is whether the PR makes sense / is consistent without these bits. Since it is working to provide MLD, I would say yes :)

(It is also missing some other pieces marked in the code, but what's missing isn't breaking things, leaking memory, etc.)

Reply:
Yeah even I am fine with having these changes as a separate PR. I saw few of the CLIs' show running present in this PR, therefore I have pointed it out so that we know it needs to be done 😄